metrics/prometheus/prometheus.go

```diff
@@ -0,0 +1,28 @@
+package prometheus
```

Comment:
No need to have it be called "prometheus". Just have a single package named `metrics`.

Reply:
Also, rather than defining all the metrics here, just define the namespace here and define the metrics where they are used.